scripts/controller.gd

```diff
@@ -5,6 +5,11 @@ enum STATES {AWAITING_ROUND, AWAITING_RESULT}
 
 const MAX_MOVES = 5
 
+var MSG_FILTER := PackedStringArray([
+	DSGMessageType.ROUND_RESULT,
+	DSGMessageType.ROUND_START,
+])
+
```

Comment:
not really convinced that this is the cleanest solution, but it does the trick for now